components/Calender/index.vue

```diff
@@ -128,14 +132,14 @@ onMounted(() => {
 const PrevMonth = () => {
     // implement this
 
-    // datesOfMonth.value = getDaysOfPrevMonth("prev");
+    datesOfMonth.value = getDaysOfPrevMonth("prev");
     // addEventindatesOfMonth();
     // giveHeightToEachRow();
 };
 const NextMonth = () => {
     // implement this
 
-    // datesOfMonth.value = getDaysOfPrevMonth("next");
+    datesOfMonth.value = getDaysOfPrevMonth("next");
     // addEventindatesOfMonth();
     // giveHeight();
 };
```

Comment:
I think we can comment on all this code because right now we are not implemented them yet
![image](https://github.com/rohit9123/compro-calender/assets/86454870/177f3b16-bd53-4162-b6db-1ad4386bf855)